crates/proof-of-sql/src/sql/proof_exprs/literal_expr.rs

```diff
@@ -57,11 +57,11 @@ impl<C: Commitment> ProofExpr<C> for LiteralExpr<C::Scalar> {
     #[tracing::instrument(name = "LiteralExpr::prover_evaluate", level = "debug", skip_all)]
     fn prover_evaluate<'a>(
         &self,
-        builder: &mut FinalRoundBuilder<'a, C::Scalar>,
+        table_length: usize,
```

Comment:
`builder.mle_evaluations.input_one_evaluation` in `verifier_evaluate` also depends on table length, so we should handle that with this change as well.